src/App.vue

```diff
@@ -203,7 +203,7 @@
             v-model="showWarning"
             :timeout="warnings[0].timeout || 2000"
             @update:model-value="dismissWarning(warnings[0])"
-            :multi-line="warnings[0].isMultiLine"
+            :multi-line="true"
```

Comment:
did you test whether this is an issue with short or long messages or messages with linebreaks?

Reply:
Does this look good enough ? 

<img width="474" alt="image" src="https://user-images.githubusercontent.com/70775490/234418259-1e13fd96-36f7-486b-b84f-0ed811b108e7.png">

Reply:
A few lines below, i set the max width of the snackbar so it will never get bigger

Reply:
overflow scroll? Or how do you enforce a max-width?

Reply:
If you text is too long then it will flow out of the application.

![image](https://user-images.githubusercontent.com/70775490/234419158-8075863f-778d-4ddf-99be-4dd8631a9407.png)

Max-width is a prop on the component

Reply:
What I could do is something like adding a scrollbar: 
<img width="350" alt="image" src="https://user-images.githubusercontent.com/70775490/234421025-97b63f80-4a50-4704-a921-486f519a0500.png">

and defining the max-height of it in a seperate css class

Reply:
seems like a better idea, doesn't it?

Reply:
Or we could use something else like after X chars its cut of wiht ...

<img width="343" alt="image" src="https://user-images.githubusercontent.com/70775490/234423351-c3e0c6ef-faa1-466c-ba77-4d01f3f16f20.png">

Reply:
But tbh when will er ever have such a Long warning message that er need a scroll bar, i think The trunking Version ja cleaner

Reply:
your call

Reply:
I am trying to look for a solution for the stacked version but currently I am facing some issues which I try to figure out

![image](https://user-images.githubusercontent.com/70775490/234568001-5b9fe62c-72ef-44d9-bbae-46a8a75b0830.png)

Reply:
I'd also recommend to stick to one color (something red)

Reply:
Currently it is possible to pass something in like "warning", "error", "info", "success" then it gets a specific color from it, so this warning system could be basically be renamed as a notification system because it has the possibilities to notify the user from error that Accor, when something successes and informational notifications

Reply:
that works for me as well

Reply:
Ok, but first ill try to first the stacking of the notifications